stripe.go

```diff
@@ -62,6 +69,9 @@ var EnableTelemetry = true
 // Key is the Stripe API key used globally in the binding.
 var Key string
 
+// ClientID is the Stripe Client ID used globally in the binding.
```

Comment:
Could you elaborate on this comment a bit by saying that: all OAuth requests will use this `ClientID` by default unless the client ID specific request's parameter is overridden.

Also, I'm not 100% convinced that we need this one — it wouldn't that much worse for users to just keep `ClientID` in their own global config somewhere and just pass it in whenever they need a Stripe OAuth requests (of which there hopefully won't be that many).

cc @remi-stripe For thoughts as well.

Reply:
Updated the comment. Again, keeping this here and in params was for convenience.